testgrid/conformance/upload_e2e.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script parses conformance test output to produce testgrid entries
+#
+# Assumptions:
+# - there is one log file and one JUnit file (true for current conformance tests..)
+# - the log file contains ginkgo's output (true for kubetest and sonobuoy..)
+# - the ginkgo output will give us start / end time, and overall success
+#
+# - the start timestamp is suitable as a testgrid ID (unique, monotonic)
+#
+# - the test ran in the current year unless --year is provided
+# - the timestamps are parsed on a machine with the same local time (zone) 
+#   settings as the machine that produced the logs
+#
+# The log file is the source of truth for metadata, the JUnit will be consumed
+# by testgrid / gubernator for individual test case results
+#
+# Usage: see README.md
+
+
+import re
+import sys
+import time
+from datetime import datetime
+import argparse
+import json
+import subprocess
+from os import path
```

Comment:
Import modules not methods

https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

Reply:
> Use import x for importing packages and modules. 
Use from x import y where x is the package prefix and y is the module name with no prefix. 
Use from x import y as z if two modules named y are to be imported or if y is an inconveniently long name.

>For example the module sound.effects.echo may be imported as follows:
from sound.effects import echo

`os.path` is the module https://docs.python.org/2.7/library/os.path.html